src/NuGet.Core/NuGet.Configuration/PackageSource/NuGetConstants.cs

```diff
@@ -24,6 +24,8 @@ public static class NuGetConstants
 
         public static readonly string FeedName = "nuget.org";
 
+        public static readonly string AddV3TrackFile = "AddV3Track.config";
```

Comment:
Call the file `NuGetOrgAdd.trk` and base the behavior on the existence of the file.